src/main/java/io/microprofile/config/builtin/converter/BigDecimalConverter.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright 2016 Microprofile.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+
+package io.microprofile.config.builtin.converter;
+
+import io.microprofile.config.spi.Converter;
+
+import java.math.BigDecimal;
+
+public class BigDecimalConverter implements Converter<BigDecimal>{
+
+	public BigDecimal convert(String value) {		
```

Comment:
The the value is null, return null. Otherwise, it either returns a big decimal or an exception.